systemtest/benchtest/expvar.go

```diff
@@ -74,3 +79,22 @@ func queryExpvar(out *expvar) error {
 	defer resp.Body.Close()
 	return json.NewDecoder(resp.Body).Decode(out)
 }
+
+// WaitUntilServerInactive blocks until one of the conditions occurs:
+// * APM Server is inactive (has no "active" events on the output buffer).
+// * HTTP call returns with an error
+// * Context is done.
```

Comment:
Yes, I agree it's a good idea to pass a context for that reason. I misread the comment as "server is deemed inactive when one of the following conditions is true", because we were previously returning `nil` when context was done. It looks good now.